app/admin/companies.rb

```diff
@@ -54,14 +54,16 @@
       tab :mq_assessments do
         panel 'Management Quality Assessments' do
           if resource.mq_assessments.empty?
-            'No Management Quality Assessments for this company yet'
+            div class: 'padding-20' do
+              'No Management Quality Assessments for this company yet'
+            end
           else
             resource.mq_assessments.latest_first.map do |a|
               panel "Assessement taken on #{a.assessment_date}", class: 'mq_assessment' do
                 attributes_table_for a do
                   row :level, &:status_description_short
                   row :publication_date do
-                    "#{a.publication_date.strftime('%B %Y')}"
+                    a.publication_date.strftime('%B %Y')
```

Comment:
we could use rails date/time formats for this
https://api.rubyonrails.org/classes/Time.html#method-i-to_formatted_s